setup.py

```diff
@@ -17,7 +17,7 @@
 
 setuptools.setup(
     name='UDASwissKnife',
-    version='0.7.1',
+    version='0.8.0',
```

Comment:
0.8.0 or 0.7.2? because what you add doesn't break all of the above to make it a major release, right?

Reply:
0.8.0 because it is a new functionality. It would be 0.7.2 if it had be a bugfix.